cohort-engine/src/test/java/com/ibm/cohort/engine/ZipStreamLibrarySourceProviderTest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.ibm.cohort.engine;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.zip.ZipInputStream;
+
+import org.hl7.elm.r1.VersionedIdentifier;
+import org.junit.Test;
+
+public class ZipStreamLibrarySourceProviderTest {
+	@Test
+	public void testLibraryFoundInZipSuccess() throws Exception {
+		try ( ZipInputStream zip = new ZipInputStream( new FileInputStream( "src/test/resources/cql/zip/breast_cancer_screening_v1_0_0_cql.zip") ) ) {
+		
+			ZipStreamLibrarySourceProvider provider = new ZipStreamLibrarySourceProvider(zip);
+			try( InputStream is = provider.getLibrarySource( new VersionedIdentifier().withId("FHIRHelpers") ) ) { 
+				assertNotNull( is );
```

Comment:
Shut me down on this one if I'm getting too pedantic, but I'd like to see some form of content verification here.
Maybe consume the stream, hash it, and compare with an expected value, or just search the stream for `FHIRHelpers`.
Asserting non null doesn't save us from bugs in the future where the wrong content is returned for whatever reason.

Reply:
Not pedantic. This was quick and dirty to improve test coverage so that I could get Matt something usable. I'll loop back and beef it up.